slugid/slugid.go

```diff
@@ -77,11 +97,7 @@ func nice() (string, error) {
 func decode(context extpoints.Context) (string, error) {
 	slug := context.Arguments["<slug>"].(string)
 
-	// validation
-	match, err := regexp.MatchString("^[A-Za-z0-9-_]{22}$", slug)
-	if err != nil {
-		return "", fmt.Errorf("Error validating slug: %s", err)
-	}
+	match := V4_SLUG_REGEXP.MatchString(slug)
```

Comment:
Would it make sense to add a comment that NICE slugs are a subset of V4 slugs (and thus match the V4 format)?